### PR TITLE
Release/0.1.0

### PR DESCRIPTION
**Pull request by adatzer**
Todo:

- [x] Test `cd` github workflow with an `-rc` release
- [x] Create docs page
- [x] Link to created page from metadata.yaml
- [x] Link to created page from README.md
- [x] Fixup additional commits

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,110 @@
...
+[![License][license-image]][license]
+[![Release][release-image]][releases]
+
+[Snowplow][snowplow] is a scalable open-source platform for rich, high quality, low-latency data collection. It is designed to collect high quality, complete behavioral data for enterprise business.
+
+## Overview
+
+This is a Google Tag Manager [custom tag template][gtm-custom-template] for implementing Ecommerce tracking using the [Snowplow JavaScript tracker v3][javascript-tracker] and its [Snowplow Ecommerce plugin][snowplow-ecommerce-plugin].
+
+It has been designed to be used with the [Snowplow v3 Settings variable template][gtm-v3-settings-variable] and can be implemented along the [Snowplow v3 tag template][gtm-v3-tag].
```
Comment (jethron) [inline README.md]: Nit: `along with` or `alongside`, not just `along`

File: template.tpl [hunk 2]
```diff
@@ -0,0 +1,2413 @@
...
+        "displayValue": "Google Analytics 4 Ecommerce"
+      },
+      {
+        "value": "uaEcommerce",
+        "displayValue": "Universal Analytics Enhanced Ecommerce"
+      }
+    ],
+    "simpleValueType": true,
+    "defaultValue": "spEcommerce",
+    "help": "Specify the Snowplow Ecommerce API to use. If you already use Google Analytics 4 ecommerce or Universal Analytics Enhanced Ecommerce to collect information about the shopping behavior of your users, you can use the respective APIs as a way to quickly implement Snowplow Ecommerce without making many changes on your current setup. However to get the full value of it, you should properly plan a full migration towards Snowplow Ecommerce API.",
```
Comment (jethron) [inline template.tpl]: I feel this should link to the [transitional API docs](https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/plugins/snowplow-ecommerce/#ga4ua-ecommerce-transitional-api) specifically just to be clear; I think without that it would be easy to think that this can do [Enhanced Ecommerce](https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/plugins/enhanced-ecommerce/) plugin tracking when it very much does not.

Comment (jethron) [reply template.tpl]: Maybe rephrase to like:

> Use the native Snowplow Ecommerce API or transitional GA4/UA ecommerce adapter APIs for existing dataLayer implementations using those formats. To get full value from the Snowplow Ecommerce plugin we recommend using the native API when possible.

File: template.tpl [hunk 3]
```diff
@@ -0,0 +1,2413 @@
...
+    "type": "GROUP",
+    "name": "snowplowGroup",
+    "displayName": "Tracking Parameters",
+    "groupStyle": "ZIPPY_OPEN",
+    "subParams": [
+      {
+        "type": "SELECT",
+        "name": "spEcomFunction",
+        "displayName": "Ecommerce Function",
+        "macrosInSelect": false,
```
Comment (jethron) [inline template.tpl]: Particularly if you've got a single ecommerce Variable it might be good to allow macros here so people can just map their events with a lookup table, rather than needing to copy the tag a bunch of times for each event.

File: template.tpl [hunk 4]
```diff
@@ -0,0 +1,2413 @@
...
+                  },
+                  {
+                    "type": 1,
+                    "string": "execute"
+                  }
+                ],
+                "mapValue": [
+                  {
+                    "type": 1,
+                    "string": "_snowplow_trackers"
```
Comment (jethron) [inline template.tpl]: Don't think we access this?

File: template.tpl [hunk 5]
```diff
@@ -0,0 +1,2413 @@
...
+            "displayName": "Advanced Tracker Settings",
+            "groupStyle": "ZIPPY_CLOSED",
+            "subParams": [
+              {
+                "type": "TEXT",
+                "name": "globalName",
+                "displayName": "Global name",
+                "simpleValueType": true,
+                "defaultValue": "snowplow",
+                "help": "The global function name to be created. \u003cstrong\u003eWarning:\u003c/strong\u003e Changing the default value requires a corresponding change to the template\u0027s \u003cstrong\u003eAccesses global variables\u003c/strong\u003e permission.",
```
Comment (jethron) [inline template.tpl]: I know it's mostly for consistency with the other templates, but do we think this is worth offering long-term?
Updating the permissions disconnects you from the Gallery version anyway, I feel like at that point we may as well just make them change a constant in the code. :shrug:

It's also the kind of thing that really should live in the Settings imo, a Tag shouldn't care about it.

File: template.tpl [hunk 6]
```diff
@@ -0,0 +1,2413 @@
...
+                  },
+                  {
+                    "type": 1,
+                    "string": "execute"
+                  }
+                ],
+                "mapValue": [
+                  {
+                    "type": 1,
+                    "string": "snowplow.apply"
```
Comment (jethron) [inline template.tpl]: Don't think we access this?

**Review by jethron — APPROVED** [review]
LGTM, great work as always!

**Comment by adatzer** [comment]
Thank you very much for reviewing @jethron !
Loved the idea to allow selecting the ecommerce function from a lookup table!
Also hardcoded the global name. You are right, it does not really worth making it configurable here.
I applied the changes based on your comments into additional commits (to be rebased before merging).
